src/app/shared/components/form-error-message/form-error-message.component.ts

```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core'
+
+@Component({
+  selector: 'sofie-form-error-message',
+  templateUrl: './form-error-message.component.html',
+  styleUrls: ['./form-error-message.component.scss'],
+})
+export class FormErrorMessageComponent {}
```

Comment:
This comment applies for both this component and `FormRequiredMark`.

I am not sure that `Form` should be in the name. Right now, this component is used within our inputs. Our inputs don't really know anything about forms, and it is possible to use our inputs without forms. There is also nothing in these components that references forms at all.

Now, if you did as suggested in another comment and did not put the error fields within our inputs, but instead added the error fields in the markup where we build our form, then the naming make much more sense. (This last part only applies for this component)

Reply:
Removed